internal/servers/controller/handlers/sessions/session_list_benchmarks_test.go

```diff
@@ -0,0 +1,107 @@
+package sessions_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/boundary/internal/db"
+	pbs "github.com/hashicorp/boundary/internal/gen/controller/api/services"
+	"github.com/hashicorp/boundary/internal/iam"
+	"github.com/hashicorp/boundary/internal/kms"
+	"github.com/hashicorp/boundary/internal/servers/controller/auth"
+	"github.com/hashicorp/boundary/internal/servers/controller/handlers/sessions"
+	"github.com/hashicorp/boundary/internal/session"
+	"github.com/hashicorp/boundary/internal/types/scope"
+	"github.com/hashicorp/boundary/testing/dbtest"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/sync/errgroup"
+)
+
+type template struct {
+	name                        string
+	sessions                    int
+	connsPerSession             int
+	authorizedSessionPercentage dbtest.AuthorizedSessionPercentage
+}
+
+func BenchmarkSessionList(b *testing.B) {
+	cases := []struct {
+		clients int
+	}{
+		{clients: 10},
+		/*
+			{clients: 100},
+			{clients: 1000},
+			{clients: 2000},
+		*/
+	}
+
+	for _, bc := range cases {
+		for _, template := range []template{{
+			name:                        dbtest.Boundary1000Sessions50ConnsPerSession50Template,
+			sessions:                    1000,
+			connsPerSession:             50,
+			authorizedSessionPercentage: dbtest.BobRatio,
```

Comment:
This seems odd, the coupling of the string const in the `dbtest` pacakge with the other variables here, it seems like this should all be defined together as part of a single struct.